scripts/release.js

```diff
@@ -10,7 +10,7 @@ const semver = require("semver")
 const R = require("ramda")
 
 const BUILD_DIR = "build/"
-const PACKAGES_DIR = "build/packages"
+const PACKAGES_DIR = "packages/build"
```

Comment:
Isn't the build folder in each package now? e.g. `packages/core/build`? I guess we'll need to refactor this script to use lerna for publishing... Or leave it as it is when shit hits the fan.

Reply:
Yes, release script is outdated, with the ci process is enough. This file changed because i've did a full text replace

```
yarn release:build
yarn version:latest
yarn publish:latest
```

Reply:
Cool. If it's possible to run these commands locally to publish new version manually, then we can remove the `release.js` file 👍

Reply:
Yes, I'll work on another branch about what we spoke this morning. Verdaccio + testing examples with verdaccio version, I'll rename release.js, to local-release.js, and i'll write there the logic.